lib/manager.js

```diff
@@ -246,21 +246,22 @@ internals.Manager.prototype._loadHelpers = function (engine) {
 
             file = Path.join(path, file);
             const stat = Fs.statSync(file);
-            if (!stat.isDirectory() &&
-                Path.basename(file)[0] !== '.') {
+            if (stat.isDirectory() || Path.basename(file)[0] === '.' ||
+                Object.keys(require.extensions).indexOf(Path.extname(file)) === -1) {
```

Comment:
Can we make this conditional multiline?
```
if (stat.isDirectory() ||
    Path.basename(file)[0] === '.' ||
    Object.keys(require.extensions).indexOf(Path.extname(file)) === -1) {
    ...
}
```

Reply:
Sure. Unfortunately, there are merge conflicts since this was left for so long.

I'll look into it.